tests-beta/android/MASVS-NETWORK/MASTG-TEST-0x22-2.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Expired Certificate Pins
+platform: android
+id: MASTG-TEST-0x22-2
```

Comment:
Could you please update each test and give it a unique ID like here: https://github.com/OWASP/owasp-mastg/tree/master/tests-beta/android/MASVS-NETWORK. This means also changing the filename accordingly.